src/lib/misc.mjs

```diff
@@ -38,6 +38,23 @@ export function isTTY() {
   return process.stdout.isTTY;
 }
 
+/**
+ * Formats an object for display.
+ * @param {any} obj - The object to format
+ * @param {object} [opts] - Options
+ * @param {boolean} [opts.json] - Whether to return a JSON string with no pretty-printing
+ * @returns {string} The formatted object
+ */
+export function formatObject(obj, opts = {}) {
```

Comment:
👍 

are there few enough usages of formatObjectForShell that we can remove the export from it so this is the only easy-to-use formatting method?

Reply:
i don't want to touch any of this because there is a story to change all this for the formatting story.